components/widgets/TemperatureCurrent/index.js

```diff
@@ -0,0 +1,49 @@
+import { temperatureUnitType } from "@/components/shapes/units";
+import { useTranslation } from "react-i18next";
+import PropTypes from "prop-types";
+import * as Styled from "./styles";
+
+const TemperatureCurrent = ({ temperature, unit = "celsius" }) => {
+  const { t, i18n } = useTranslation();
+  const { language = "en" } = i18n;
+
+  const formatTemperature = (value) => {
+    const formatter = new Intl.NumberFormat(language, {
+      style: "unit",
+      unit,
+      maximumFractionDigits: 0,
+    });
+
+    return formatter
+      .formatToParts(value)
+      .map(({ type, value }) => {
+        if (type === "unit") {
+          return value.replace(/[a-z]/gi, "");
+        }
+
+        return value;
+      })
+      .join("");
+  };
```

Comment:
add a formatter on top of the JavaScript `Intl` lib to remove the "F" or "C" but preserve the degrees symbol and any other formatting applied